koku/reporting/provider/all/openshift/models.py

```diff
@@ -577,7 +577,7 @@ class Meta:
     # It is used to divide cost evenly among projects
     shared_projects = models.IntegerField(null=False, default=1)
 
-    source_uuid = models.UUIDField(unique=False, null=True)
+    source_uuid = models.ForeignKey("api.Provider", on_delete=models.CASCADE, db_column="source_uuid", null=True)
```

Comment:
Why the foreign key here?

Reply:
@adberglund 
To get the table data to delete when the source is deleted. Else we run the risk of stale data inflating totals. (in testing, at least)

Reply:
Will link with bill id instead.